src/natcap/invest/urban_flood_risk_mitigation.py

```diff
@@ -493,17 +534,59 @@ def _write_summary_vector(
             pixel_count = runoff_ret_vol_stats[feature_id]['count']
             if pixel_count > 0:
                 damage_sum = damage_per_aoi_stats[feature_id]
-                target_feature.SetField('aff_bld', damage_sum)
+                target_feature.SetField('aff.bld', damage_sum)
+
+                # This is the service.built equation.
                 target_feature.SetField(
-                    'serv_bld', damage_sum * float(
-                        runoff_ret_vol_stats[feature_id]['sum']))
+                    'serv.blt', service_built_stats[feature_id])
+

```

Comment:
This comment touches a few different places, but I'll put it here near the old `serv_bld` math. I kinda expected the `serv.blt` results to change because we have this new `_calculate_service_built` function which uses stats from `runoff`, whereas the old `serv_bld` used stats from `runoff_ret_vol`. But I see the test results have not changed, so is it actually possible to calculate `serv.blt` from the pre-existing `runoff_ret_vol_stats`? 

Also, can I get that on whole wheat with mayo?

Reply:
🥪 coming right up!

Yes, I expected the `serv_bld` results to have changed as well and had written this off as a nuance of the sample data used.  It turns out that the original approach (`damage_sum * runoff_ret_vol_stats[feature_id]['sum']`) was, in fact, correct.  It was just expressed in the UG in a way that was an algebraic step or two away from what was implemented here before, and was mathematically equivalent to what I had expressed in `_calculate_service_built`.

So I've reverted this back to the simpler, older form in f4a5044a and updated the UG chapter to reflect this.  I think the more abbreviated version of this in the UG better expresses what's happening, anyways.